recipes/pycutest/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pycutest" %}
+{% set version = "1.0" %}
+{% set archdefs_version = "v2.0.4" %}
+{% set sifdecode_version = "v2.0.3" %}
+{% set cutest_version = "v2.0.3" %}
+{% set sif_version = "v0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycutest-{{ version }}.tar.gz
+    sha256: 7543dc796e3279a260a362c559807e4f099398f27656837a7aad68b58eb81fe6
+  - url: https://github.com/ralna/ARCHDefs/archive/refs/tags/{{ archdefs_version }}.zip
+    sha256: 87d6603c92705b2b78f0a00ecb1c0ce6f63e4a3ce460552b47ef872b1cd66b46
+    folder: archdefs
+  - url: https://github.com/ralna/SIFDecode/archive/refs/tags/{{ sifdecode_version }}.zip
+    sha256: cfc0a5a82caada92da29b97cc26603f660bf1ddde0478cabf14f5d4740a0f385
+    folder: sifdecode
+  - url: https://github.com/ralna/CUTEst/archive/refs/tags/{{ cutest_version }}.zip
+    sha256: 537f3ab5688ef92b102f8f23f2da125f1e798beeb808e261aec4dee02b4eb1aa
+    folder: cutest
+  - url: https://bitbucket.org/optrove/sif/get/{{ sif_version }}.zip
+    sha256: a23214f2fb285860caa27bf05a767d75a441574ab126d60f004f9318a5911b5b
+    folder: mastsif
+
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - sed
+    - make
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.11
+    - python
+    - scipy >=0.17
+    - gcc_linux-64
+    - gfortran_linux-64
```

Comment:
These shouldn't be in `run`.
```suggestion
```

Reply:
Once installed, pycutest may compile some pieces of code, this is why the recipe installs compilers for the runtime.

Reply:
The you should put in here the `compiler` macros and not the rendered names.

Reply:
Sure, I have no idea why I didn't use the macros!